tools/integration_tests/util/operations/file_operations.go

```diff
@@ -50,9 +50,9 @@ func copyFile(srcFileName, dstFileName string, allowOverwrite bool) (err error)
 
 	var destination *os.File
 	if allowOverwrite {
-		destination, err = os.OpenFile(dstFileName, os.O_WRONLY|os.O_CREATE|syscall.O_DIRECT|os.O_TRUNC, FilePermission_0600)
```

Comment:
why is this syscall.O_DIRECT and os.O_DIRECT?

Also why was O_DIRECT required here?

Reply:
```
Currently we are using linux command to clear the cache, instead we should replace with the O_DIRECT flag while reading any file.

os.OpenFile(name, syscall.O_DIRECT)
```

@raj-prince suggested this before.

But while executing some operations it is throwing invalid argument error.
I raised question on yaqs, they suggested to remove this flag `filesystem doesn't support this where you are reading the file.`